components/Deck/ContentModulesPanel/ContributorsPanel/ContributorItem.js

```diff
@@ -10,7 +10,7 @@ class ContributorItem extends React.Component {
                     <div className="ui image">
                         <UserPicture picture={ this.props.data.picture }
                             username={ this.props.data.username } link={ false }
-                            private={ false } size={ 'mini' } avatar={ true } width= { 24 } bordered={ false } alt={'""'}/>
+                            private={ false } size={ 'mini' } avatar={ true } width= { 24 } bordered={ false } alt={'""'} aria-hidden="true"/>
```

Comment:
I don't think this has the desired effect, from what I can tell by inspecting the resulting html. If we want this for all `UserPicture` instances, this change should be included in `UserPicture` file.

Reply:
moved aria-hidden to Userpicture component and remove role=presentation (as presentation role not as well supported by screen readers)